### DISCUSSION TITLE
Can I change the line spacing option only for block quotes when I convert from markdown to PDF?

### DISCUSSION THREAD

**Discussion by kcho17** [jgm/pandoc#9841] [Q&A] [answered]
Is there any way that I can change the line spacing (or line stretching) only for the block quotes when converting from markdown to pdf.

**Comment by jgm** [comment]
Certainly. Create a small tex file, `quotespace.tex`, containing:
```tex
\usepackage{setspace}
\AtBeginEnvironment{quote}{\setstretch{1.2}}
```
Then add option `--include-in-header quotespace.tex` when you invoke pandoc to produce a PDF. (Change 1.2 to your desired value.)

**Comment by jgm** [comment, marked as answer]
Alternatively you can add to your metadata:
````
header-includes:
- |
    ```{=latex}
    \usepackage{setspace}
    \AtBeginEnvironment{quote}{\setstretch{1.2}}
    ```
````
And then you don't need a special option or file.
This would also work:
````
header-includes: |
  ```{=latex}
  \usepackage{setspace}
  \AtBeginEnvironment{quote}{\setstretch{1.2}}
  ```
````

**Reply by kcho17** [reply]
Thanks! This works!
